C24_WMDE_Desktop_DE_07

Sperling-0 opened the pull request:
- Both banners:
        - are based on the control banner of desktop-de-06.

- Variant banner:
       - The variant banner is revived from [desktop-de-02](https://de.wikipedia.org/?banner=B24_WMDE_Desktop_DE_02_var&devMode).
       - The variant banner's headline is changed to "Wikipedia ist unverkäuflich".

- Use swatches in the fallback banner
- Change BIC
- Inline UoF infographics

Ticket: https://phabricator.wikimedia.org/T375603

Sperling-0 committed:
Prepare campaign for C24_WMDE_Desktop_DE_07

Ticket: https://phabricator.wikimedia.org/T375603

Sperling-0 committed:
Prepare CTRL for C24_WMDE_Desktop_DE_07

CTRL is  based on the control banner of desktop-de-06.

Ticket: https://phabricator.wikimedia.org/T375603

Sperling-0 committed:
Implement VAR for C24_WMDE_Desktop_DE_07

- It is revived from desktop-de-02
- The variant banner's headline is changed to "Wikipedia ist unverkäuflich"
- Use correct css variable names
- Remove the old banner colours setup

Ticket: https://phabricator.wikimedia.org/T375603

Abban commented on banners/desktop/C24_WMDE_Desktop_DE_07/styles/colors.ts:
You can delete this

Abban commented on banners/desktop/C24_WMDE_Desktop_DE_07/styles/fallback_colors.ts:
And delete this too

Abban commented on test/banners/desktop/components/FallbackBanner.spec.ts:
```diff
@@ -1,6 +1,6 @@
 import { afterEach, beforeEach, describe, test, vi } from 'vitest';
 import { mount, VueWrapper } from '@vue/test-utils';
-import FallbackBanner from '@banners/desktop/C24_WMDE_Desktop_DE_03/components/FallbackBanner.vue';
+import FallbackBanner from '@banners/desktop/C24_WMDE_Desktop_DE_07/components/FallbackBanner.vue';
```
Oh right, we've been forgetting all about the fallback banner!

Sperling-0 commented:
⚠️ Rebase this PR from latest main.

To get the latest UoF working:

- For Desktop banners: `BannerCtrl.vue`, `BannerVar.vue`, `FallbackBanner.vue` needs to be updated. (If it is DE desktop include DE infographic and vice versa)
- For Mobile banners: `BannerCtrl.vue`, `BannerVar.vue` needs to be updated. (If it is DE mobile include DE infographic and vice versa)

Sperling-0 committed:
Use swatches in the fallback banner

- fix transparent banner background

Ticket: https://phabricator.wikimedia.org/T375603

Sperling-0 committed:
Change BIC

- Request: https://mattermost.wikimedia.de/swe/pl/x1rz5qd6n7fu9gxf7wbo58kn9o

Ticket: https://phabricator.wikimedia.org/T377589

Sperling-0 committed:
Inline UoF infographics

- https://github.com/wmde/fundraising-banners/pull/563
- Update content repo

Ticket: https://phabricator.wikimedia.org/T375848

Sperling-0 commented:
> ⚠️ Rebase this PR from latest main.
> 
> To get the latest UoF working:
> 
>     * For Desktop banners: `BannerCtrl.vue`, `BannerVar.vue`, `FallbackBanner.vue` needs to be updated. (If it is DE desktop include DE infographic and vice versa)
> 
>     * For Mobile banners: `BannerCtrl.vue`, `BannerVar.vue` needs to be updated. (If it is DE mobile include DE infographic and vice versa)

✅ This branch is rebased to latest main. Has 'Inline UoF infographics' changes according to https://github.com/wmde/fundraising-banners/pull/563
